.run/extrusion-create.run.xml

```diff
@@ -4,6 +4,11 @@
     <module name="baremaps-cli" />
     <option name="PROGRAM_PARAMETERS" value="workflow execute --file workflow.json" />
     <option name="WORKING_DIRECTORY" value="$PROJECT_DIR$/examples/extrusion" />
+    <extension name="software.aws.toolkits.jetbrains.core.execution.JavaAwsConnectionExtension">
```

Comment:
This section has probably been automatically added by IntelliJ.

Reply:
Addressed in https://github.com/apache/incubator-baremaps/pull/658/commits/aac687ed7e1c6bc411992610b3964ad2a750842d.
Aligned all `run.xml` without this section as some had it previously and my IDE added it to some others.